src/PublixScraper.py

```diff
@@ -24,7 +24,7 @@ def scrape_publix():
     options.add_experimental_option("prefs", {"profile.default_content_setting_values.geolocation": 2})
 
     # Installs driver depending on browser
-    driver=webdriver.Chrome(service=Service(ChromeDriverManager().install()),options=options)
+    driver=webdriver.Chrome(service=Service(ChromeDriverManager(version='114.0.5735.90').install()),options=options)
```

Comment:
what happens when there's a new version of Chrome? Will this not work anymore?

Reply:
I think it is a version of the chrom driver manager, not chrome. I am not fully sure but I think it is more having to do with the people who make this package rather than our version of chrome

Reply:
The whole thing was there were updates that did not align with the new drivers or something and the manager package

Reply:
ah I see, we just need to make sure to keep an eye on this